### PR TITLE
Argparse, formatting and Python3

### PR DESCRIPTION
**Pull request by grebsai**
Since the optparse module is deprecated, I changed it to argparse - the new module.
I also added some formatting and minor Python3 related adjustments.

### CODE REVIEW

**Comment by Fenixin** [comment]
Thanks a lot for pull, I will review and merge soon (tm).

File: regionfixer.py [hunk 1]
```diff
@@ -573,17 +554,12 @@ def main():
     try:
         freeze_support()
         value = main()
- 
-    except WrongOption as e:
```
Comment (Fenixin) [inline regionfixer.py:577]: Is there a reason why you removed this return value?

Comment (grebsai) [reply regionfixer.py:577]: `BadOptionError` comes from the old module `optparse` and `WrongOption` inherits from this class. Also the `NewOptionParser` inherits from the `OptionParser` from the old module `optparse`. Therefore both are no longer used and in my opinion are not necessary.

Comment (Fenixin) [reply regionfixer.py:577]: I would really like to keep that return code, could you find a way to add it again using argparse?

**Comment by Fenixin** [comment]
Thanks again for your code. I have merged this into `bugfix` and added some code to use exit values as I wanted. It will be merged with master in some time.

**Pull request closed without merging**